GameServerLib/GameObjects/AttackableUnits/AttackableUnit.cs

```diff
@@ -1190,7 +1190,11 @@ public void AddBuff(IBuff b)
                         _game.PacketNotifier.NotifyNPC_BuffAdd2(b);
                     }
                     // Activate the buff for BuffScripts
-                    b.ActivateBuff();
+                    for(int i = 0; i < b.MaxStacks; ++i)
```

Comment:
MaxStacks is meant to be used inside each buff script for specific stacking behavior, and the only buffs which stack that should be activated multiple times are those with a `BuffAddType` of `STACKS_AND_OVERLAPS` (and that functionality is already implemented).

Reply:
Although, in order to OnActivate get properly triggered multiple times, you have to give multiple calls or adding 1 stack.
OnActivate gets triggered only once regardless of how many buffs you're adding.
On other note, OnDeactivate also gets triggered only once regardless of many stacks the buff has.

Maybe we should have a few extra Functions in BuffScripts?
 - OnAddStack(int stacks)
 - OnRemoveStack(int stacks)

Reply:
Currently we have `OnAllowAddBuff` which should satisfy `OnAddStack`, but since we don't have an equivalent for `OnRemoveStack` it should be fine to add.